oauth2client/flask_util.py

```diff
@@ -47,11 +58,8 @@ def example():
         # Or, you can access the credentials directly
         credentials = oauth2.credentials
 
-
-    @app.route('/info')
-    @oauth2.required
-    def info():
-        return "Hello, {}".format(oauth2.email)
+If you want credentials to be optional for a view, you can leave the decorator
+off and use :meth:`UserOAuth2.has_credentials` to check::
```

Comment:
Want to use `code-block:: python` instead? (Maybe since in a docstring it is automatic?)